easybuild/easyblocks/c/condaenv.py

```diff
@@ -1,129 +0,0 @@
-##
-# Copyright 2009-2016 Ghent University
-#
-# This file is part of EasyBuild,
-# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
-# with support of Ghent University (http://ugent.be/hpc),
-# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
-# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
-# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
-#
-# http://github.com/hpcugent/easybuild
-#
-# EasyBuild is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation v2.
-#
-# EasyBuild is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
-##
-"""
-EasyBuild support for building and installing conda environments via conda env
-create -p, implemented as an easyblock @author: Jillian Rowe (New York
-University Abu Dhabi)
-"""
-
-import os
-import shutil
-import stat
-
-import easybuild.tools.environment as env
-from easybuild.framework.easyblock import EasyBlock
-from easybuild.framework.easyconfig import CUSTOM
-from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.filetools import rmtree2
-from easybuild.tools.run import run_cmd
-from easybuild.easyblocks.anaconda import initialize_conda_env, set_conda_env
-
-class EB_CondaEnv(EasyBlock):
-    """Support for building/installing environments using conda env. You must use conda-env >= 2.5.2"""
-
-    @staticmethod
-    def extra_options(extra_vars=None):
-        """Extra easyconfig parameters specific to EB_CondaEnv easyblock."""
-        extra_vars = EasyBlock.extra_options(extra_vars)
-        extra_vars.update({
-            'environment_file': [None, "Environment.yml file. Either add a 'environment.yml' to your sources, or specify the full path here.", CUSTOM],
-            'post_install_cmd': [None, "Commands after install: pip install, cpanm install, etc", CUSTOM],
-            'pre_install_cmd': [None, "Commands before install: setting environment variables, etc", CUSTOM],
```

Comment:
@jerowe note: I dropped these, since they have equivalent generic easyconfig parameters:

* rather than `post_install_cmd`, you can use `postinstallcmds = [...]`
* rather than `pre_install_cmd`, you can use `preinstallopts = "... && "`